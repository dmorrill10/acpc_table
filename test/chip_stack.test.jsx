import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';
const ChipStack = require('../dist/chip_stack.js');

describe('ChipStack', () => {
  it('shows up for amounts greater than zero', () => {
    const patient = <ChipStack amount={9001}/>;
    const wrapper = shallow(patient);
    expect(wrapper.text()).toEqual('9001');
    expect(wrapper.hasClass('chip-stack')).toEqual(true);
    expect(renderer.create(patient).toJSON()).toMatchSnapshot();
  });
  it('does not show up for amounts less than 1', () => {
    const patient = <ChipStack amount={0}/>;
    const wrapper = shallow(patient);
    expect(wrapper.type()).toEqual(null);
  });
});
