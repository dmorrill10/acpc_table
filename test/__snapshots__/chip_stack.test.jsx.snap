exports[`ChipStack shows up for amounts greater than zero 1`] = `
<div
  className="chip-stack">
  <div
    className="amount">
    9001
  </div>
</div>
`;
